x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/external/http/sender/HttpRequestExecutorServiceTests.java

```diff
@@ -1,264 +0,0 @@
-/*
```

Comment:
Sigh... most of the tests here didn't change much except for aligning with the new constructor arguments. I guess it didn't pick up that this file was renamed 🤦‍♂️